src/menu/DatasetCard.tsx

```diff
@@ -0,0 +1,78 @@
+import React from 'react';
+import {Card, Nav, Tab, Row} from 'react-bootstrap';
+import {INamedSet, ENamedSetType, RestBaseUtils, RestStorageUtils} from 'tdp_core';
+import {NamedSetList, useAsync, IStartMenuDatasetDesc} from 'ordino';
+import {UserSession} from 'phovea_core';
+import {DatasetSearchBox} from './DatasetSearchBox';
+
+
+
+
+export default function DatasetCard({name, headerIcon, tabs, viewId, datasource}: IStartMenuDatasetDesc) {
+  const subTypeKey = 'species';
+
+  const loadPredefinedSet = React.useMemo(() => {
+    return () => RestBaseUtils.getTDPData(datasource.db, `${datasource.base}_panel`)
+      .then((panels: {id: string, description: string, species: string}[]) => {
+        return panels
+          .map(function panel2NamedSet({id, description, species}): INamedSet {
+            return {
+              type: ENamedSetType.PANEL,
+              id,
+              name: id,
+              description,
+              subTypeKey,
+              subTypeFromSession: false,
+              subTypeValue: species,
+              idType: ''
+            };
+          });
+      });
+  }, [datasource.db, datasource.base, datasource.idType]);
+
+  const loadNamedSets = React.useMemo(() => {
+    return () => RestStorageUtils.listNamedSets(datasource.idType);
+  }, [datasource.db, datasource.base, datasource.idType]);
```

Comment:
```suggestion
  }, [datasource.idType]);
```

Can we remove these since they are not used inside the function?